Doc/library/os.rst

```diff
@@ -4538,7 +4538,7 @@ The following functions take a process status code as returned by
 :func:`system`, :func:`wait`, or :func:`waitpid` as a parameter.  They may be
 used to determine the disposition of a process.
 
-.. function:: WCOREDUMP(status)
+.. function:: WCOREDUMP(status, /)
```

Comment:
Looking at https://github.com/python/cpython/blob/1009bf18b38a8d36298575191dd8fdf43f8f9097/Modules/clinic/posixmodule.c.h#L4282-L4318 when the file was first made the markers were already in place, so maybe there's some reason for it.

Separately, could you check whether you're able to run [`os.plock(op=None)`](https://docs.python.org/3/library/os.html#os.plock) to verify the keyword can't be used? I gave all the functions a quick run to double-check but like `chflags`, I'm not able to run `plock` for some reason.

Reply:
Actually, seeing that the clinic def

https://github.com/python/cpython/blob/f0bf7956e60b452208f279146f928d71eb0aa11b/Modules/clinic/posixmodule.c.h#L3688-L3692

has the position-only marker, then regardless of whether it can be confirmed, adding the `plock` change would arguably be strictly better than the current docs. In that case, tracking this particular problem can go into some other issue instead, and I'd reckon that the change can go ahead anyways, if there aren't any other problems with the PR.